src/main/scala/viper/gobra/ast/frontend/Ast.scala

```diff
@@ -423,15 +425,15 @@ case class PGreater(left: PExpression, right: PExpression) extends PBinaryExp
 
 case class PAtLeast(left: PExpression, right: PExpression) extends PBinaryExp
 
-case class PAdd(left: PExpression, right: PExpression) extends PBinaryExp
+case class PAdd(left: PExpression, right: PExpression) extends PBinaryExp with PNumExpression
```

Comment:
I just want to point out that we have statements with imlicit arithmetic operation, e.g. x += 1

Reply:
It seems to me that this does not affect how we type-check integer expressions. Do you see any problem that this may pose?

Reply:
I just wanted to point it out. Since constants cannot be reassigned, it probably does not matter.